### PR TITLE
How To Define a Conference Doc

### PR DESCRIPTION
**Pull request by meatballs**
For now, this PR is simply to allow comment on the WIP.

### CODE REVIEW

**Comment by meatballs** [comment]
I'm just working on some stuff for it at the moment. Let me finish that off and then we can take a look at it.

**Comment by drvinceknight** [comment]
Cool :)👍

File: docs/howto/define_a_conference.rst [hunk 1]
```diff
@@ -126,7 +126,31 @@ for PyConUK 2016 in a :download:`YAML file <pyconuk-2016-talks.yml>`
...
 
-.. todo:: Section describing how to define unavailability for people
+When organising any conference, it is common that speakers might be unavailable
+to attend on certain days or for certain time periods.
+
+Although we don't have the real information from PyCon UK 2016, for this
+example, we can create some fictitious data. Let's say that Alex Chan
+was not available on either the Friday or the Sunday morning::
+
+    >>> yaml_speaker_unvailability =  """
```
Comment (alexwlchan) [inline docs/howto/define_a_conference.rst]: /s/unvailability/unavailability

File: docs/howto/define_a_conference.rst [hunk 2]
```diff
@@ -201,6 +225,22 @@ And also the file containing the talks::
                'for a fully real time experience'}]
 
 
+Finally, the unavailability and clashes::
+
+    >>> speaker_unvailability = yaml.load(yaml_speaker_unvailability)
```
Comment (alexwlchan) [inline docs/howto/define_a_conference.rst]: ditto above

**Comment by meatballs** [comment]
OK. That's enough for now!! Feel free to pick that to bits....

**Comment by meatballs** [comment]
This will break slightly if #77 goes in

**Pull request merged**